src/scripts/components/VideoListItem.js

```diff
@@ -76,6 +107,13 @@ class VideoListItem extends Component<Props, void> {
   }
 
   handleClick () {
+    const form = document.getElementById('video-form')
+    const offsetTop = form.offsetTop - 20
+    if (form) {
+      if (offsetTop > window.innerHeight / 2) {
+        document.getElementById('root').scrollTo(0, offsetTop)
+      }
+    }
```

Comment:
This works, but there is a more react-way to get references to actual dom nodes 😄 (requires no hard-coded IDs).

Take a look at [this line](https://github.com/Paratii-Video/paratii-portal/blob/460fa75750281b9b66a0ad93a4d92dd3e9c72066/src/scripts/components/widgets/VolumeBar.js#L107) in `VolumeBar.js`. Here, we take advantage of `styled-components`'s [innerRef](https://github.com/styled-components/styled-components/blob/master/docs/tips-and-tricks.md#refs-to-dom-nodes) property. This provides a mechanism for us to access the [ref property](https://reactjs.org/docs/refs-and-the-dom.html) on the underlying react elements.

Long story short: `ref/innerRef` are properties that take a callback. This callback is invoked with an `HTMLElement` that you can then cache/store somewhere. `VolumeBar.js` is a good example.

Let me know if you have any questions about how this should work!